docs/settings/logs-ui-settings.asciidoc

```diff
@@ -0,0 +1,39 @@
+[role="xpack"]
+[[logs-ui-settings-kb]]
+=== Logs UI Settings in Kibana
+++++
+<titleabbrev>Logs UI Settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use Logs UI. It is enabled by default.
+
+[float]
+[[general-logs-ui-settings-kb]]
+==== General Logs UI Settings
+
+xpack.infra.enabled:: Set to `false` to disabled the Logs and Infrastructure UI plugin {kib}. Defaults to
```

Comment:
Same comments as before.

I would not duplicate the content because it's likely to get out of sync. Instead I would either:

-  use a shared file and use an asciidoc `include` to pull in the shared file, or
- put the descriptions in one place and use a link to point to the settings. So you could say something like:
"The Logs UI shares settings with the Infrastructure UI. See <<general-infra-ui-settings-kb>>." (Or point to the top of the topic if all the settings will be shared.)

I don't like to force users to jump around. On the other hand, using `include` statements makes it harder for folks to maintain the content.